README

```diff
@@ -32,18 +32,18 @@ for visually impaired people and people with special educational needs.
 On GitHub, the following files are enclosed.
 
   README                        - This file.
-  axessibility.sty                - The axessibility package.
```

Comment:
"This work consists of the main source files axessibility.**dtx** and axessibility.lua,
% and the derived files
%   axessibility.**ins**"
Questi due file (".dtx" e ".ins") in realtà non ci sono. Li rimuovo dalla documentazione?